src/sentry/issues/grouptype.py

```diff
@@ -70,74 +88,78 @@ class ErrorGroupType(GroupType):
     slug = "error"
     description = "Error"
     category = GroupCategory.ERROR.value
-    ignore_limit = 0
+
+
+# used as an additional superclass for Performance GroupType defaults
+class PerformanceGroupTypeDefaults:
+    noise_config = NoiseConfig()
```

Comment:
don't really have a better solution to have default be `None` for all issue types and then have a default `NoiseConfig()` for performance issue types. happy to change it if anyone's aware of something better